### DISCUSSION TITLE
Use of "Go to" option for nodes/edges.

### DISCUSSION THREAD

**Discussion by Abhay0294** [eclipse-glsp/glsp#1017] [Q&A]
Hi Team,

I have been trying to understand the usage of "Go to" option, which we get on right clicking on node(or group of nodes) or edges(or group of edges). In my case it is always disabled

Requesting you to please help me in understanding its use-case scenarios.

![image](https://github.com/eclipse-glsp/glsp/assets/57217189/5cf69cf5-2079-42b1-9c29-9665abc4e7e2)

**Comment by martin-fleck-at** [comment]
Hi @Abhay0294, the "two" entries that you mention are coming from the [MarkerNavigatorContextMenuItemProvider](https://github.com/eclipse-glsp/glsp-client/blob/4e100359fa5e72415368d0670906e1d03c1d6b05/packages/client/src/features/validation/marker-navigator.ts#L203) contributed  by the [markerNavigatorContextMenuModule](https://github.com/eclipse-glsp/glsp-client/blob/4e100359fa5e72415368d0670906e1d03c1d6b05/packages/client/src/features/validation/di.config.ts#L50). The reason those entries are disabled are because your nodes do not have any markers (e.g., issues like warnings, errors, info). If you do not want them to be present you can avoid integrating the 'markerNavigatorContextMenuModule'.

**Reply by Abhay0294** [reply]
Thanks a lot @martin-fleck-at for the response. I'll try as per your suggestion.
